### PR TITLE
config: add important cdc configs to server_configs

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add some cdc configs to the `server_configs` part so that users know they can configure cdc para in this part as well, besides `cdc_servers`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/12705
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: config-templates/complex-cdc.yaml [hunk 1]
```diff
@@ -43,6 +43,11 @@ server_configs:
   cdc:
     # capture-session-ttl: 10
     # sorter.sort-dir: "/tmp/cdc_sort"
+    # gc-ttl: 86400
+    deploy_dir: "/tidb-deploy/cdc-8300"
+    data_dir: "/tidb-data/cdc-8300"
+    log_dir: "/tidb-deploy/cdc-8300/log"
```
Comment (overvenus) [inline config-templates/complex-cdc.yaml:49]:
```suggestion
```

I think we can only keep cdc specific configs.

**Review by overvenus — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**